schema.sql

```diff
@@ -1,9 +1,15 @@
 /* Database schema to keep the structure of entire database. */
+
 CREATE TABLE animals (
-    id INT BIGSERIAL NOT NULL PRIMARY KEY,
+    id INT,
     names VARCHAR(100) NOT NULL,
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL
-);
\ No newline at end of file
+    weight_kg DECIMAL,
+
+);
+
+-- Create column species
+ALTER TABLE animals ADD species TEXT;
```

Comment:
- [ ] in this column you need to use VARCHAR instead of text because you will be able to set the max length of the value.

Reply:
Thank you;